service/api_patch.go

```diff
@@ -191,6 +193,16 @@ func (as *APIServer) updatePatchModule(w http.ResponseWriter, r *http.Request) {
 		as.LoggedError(w, r, http.StatusInternalServerError, err)
 		return
 	}
+	if formatted {
```

Comment:
an alternative to passing `formatted` from the client could be to try `GetPatchSummariesByCommit` and set summaries if there's no error, similar to what this PR does for patches later.

Reply:
Well, the only reason I'm writing summaries normally and then again if formatted is because of the migration. In the future, because I'm assuming that this gets called by non-commit queue things, I only want to do lines 191-195 if it's not commit queue, and only want to do 196-202 if it is commit queue (rather than try to read the same thing twice). I could put a comment here as a reminder to refactor?

Reply:
Actually, I can check if isMboxPatch if that works for you.